### PR TITLE
Odd behaviour of libz decoding of cover_default.png at TracksTreeMode…

### PR DESCRIPTION
**Pull request by WaylonR**
…l::addCoverArt bypassed by using the SVG version

Using the PNG version, caused the UI/Thread 1 to lock up.

### CODE REVIEW

File: res/mixxx.qrc [hunk 1]
```diff
@@ -3,6 +3,7 @@
         <file>../LICENSE</file>
         <file>images/ic_mixxx_window.png</file>
         <file>images/library/cover_default.png</file>
```
Comment (Be-ing) [inline res/mixxx.qrc]: remove the PNG image here and delete the PNG file

File: res/mixxx.qrc [hunk 2]
```diff
@@ -3,6 +3,7 @@
...
+        <file>images/templates/cover_default.svg</file>
```
Comment (Be-ing) [inline res/mixxx.qrc]: move this to `images/library` instead of `images/templates`

**Review by Be-ing — CHANGES_REQUESTED** [review]

**Comment by Be-ing** [comment]
Before we can merge your code, please [sign the Mixxx Contributor Agreement](https://docs.google.com/forms/d/e/1FAIpQLScC9QG327sjLL0eWftmfGUasxFFLxg6LCXJ2xHDYRzFIRqyiw/viewform?formkey=dEpYN2NkVEFnWWQzbkFfM0ZYYUZ5X2c6MQ) and comment here when you have. In a nutshell, it allows us to distribute your work under the GPL as well as the Mac App Store which has terms that conflict with the GPL (refer to the link for the full legalese).

File: res/mixxx.qrc (hunk 1 above)
Comment (WaylonR) [reply res/mixxx.qrc]: would require all other instances to be replaced with the svg file. Suppose its doable.

**Comment by WaylonR** [comment]
Signed.

**Comment by Be-ing** [comment]
For the future, please use shorter text for your short commit message. You can explain more details in the rest of the commit message.

**Comment by Be-ing** [comment]
Looks good to me, thanks!

**Pull request merged**